source/managing_account/04_registering_applications/registering_applications.rst

```diff
@@ -0,0 +1,15 @@
+Registering applications
```

Comment:
Add a link to the documentation to use these applications from the APIs? E.g: https://help.opendatasoft.com/apis/ods-search-v2/#using-oauth2-authorization